ravenframework/Optimizers/GeneticAlgorithm.py

```diff
@@ -714,16 +1142,26 @@ def checkConvergence(self, traj, new, old):
       @ Out, any(convs.values()), bool, True of any of the convergence criteria was reached
       @ Out, convs, dict, on the form convs[conv] = bool, where conv is in self._convergenceCriteria
     """
-    convs = {}
-    for conv in self._convergenceCriteria:
-      fName = conv[:1].upper() + conv[1:]
-      # get function from lookup
-      f = getattr(self, f'_checkConv{fName}')
-      # check convergence function
-      okay = f(traj, new=new, old=old)
-      # store and update
-      convs[conv] = okay
-
+    if not self._isMultiObjective:
+      convs = {}
+      for conv in self._convergenceCriteria:
+        fName = conv[:1].upper() + conv[1:]
+        # get function from lookup
+        f = getattr(self, f'_checkConv{fName}')
+        # check convergence function
+        okay = f(traj, new=new, old=old)
+        # store and update
+        convs[conv] = okay
+    else:
+      convs = {}
+      for conv in self._convergenceCriteria:
+        fName = conv[:1].upper() + conv[1:]
+        # get function from lookup
+        f = getattr(self, f'_checkConv{fName}')
+        # check convergence function
+        okay = f(traj, new=new, old=old)
+        # store and update
+        convs[conv] = okay
     return any(convs.values()), convs
```

Comment:
@joshua-cogliati-inl, this is exactly repeated for single and multi-objective. There must be something wrong. I will take a look at it, but please do as well.

Reply:
Looks like you fixed it. :)